### DISCUSSION TITLE
Go to end of line command

### DISCUSSION THREAD

**Discussion by RGBCube** [helix-editor/helix#8756] [Q&A] [answered]
I'd like to assign `D` to `v gl esc`, but I can't, since I couldn't find the command for `gl`:

```nix
programs.helix.settings.keys.normal.D = [
	"select_mode"
	???
	"esc"
];
```

What could I put in `???`?

**Comment by pascalkuthe** [comment, marked as answer]
the keymap is simply `extend_to_line_end`. With a couple exceptions selectmode is not special and just a different keymap. So you don't need to enter select mode (esc is also not a valid keymap btw.)

**Reply by RGBCube** [reply]
Is there a builtin shortcut for this?

**Reply by pascalkuthe** [reply]
well gl in select mode is mapped to that. Like I said select mode for the most part just changes the keymap
